src/modules/jobs/dto/create-job.dto.ts

```diff
@@ -100,6 +100,7 @@ export class CreateJobDto {
   })
   hardRequirements?: string;
 
+  @IsOptional()
   @IsNotEmpty()
```

Comment:
Aqui no caso o campo poderia ser definido como vazio correto?

Reply:
Na verdade não. Ele é opcional, entretanto, se for enviado, não pode ser vazio.